MaxText/configs/base.yml

```diff
@@ -14,6 +14,9 @@
 
 # This sentinel is a reminder to choose a real run name.
 # If there is already a checkpoint under this run, that checkpoint will auto-resume.
+
+# we need to port this script to the pytorch model
```

Comment:
The logical axis rules dictate how different tensors are sharded (what axis are sharded via DP or FSDP or TP). Sharding is necessary for PyTorch, altho I think it would be okay if it's not configurable and just uses the Maxtext default (which will still take significant work that Mohit is more familiar with)